.github/workflows/pr.yml

```diff
@@ -30,12 +30,16 @@ jobs:
     steps:
     - uses: actions/checkout@v2
       name: Checkout che-theia source code
+      with:
+        fetch-depth: "0"
```

Comment:
why we need this input?

Reply:
It is needed for correct fetching tags in current action: https://github.com/lerna/lerna/issues/2542